flathunt.py

```diff
@@ -20,16 +22,45 @@
 __email__ = "harrymcfly@protonmail.com"
 __status__ = "Production"
 
+def is_current_time_between(time_from, time_till):
```

Comment:
I feel like it would be neater to have the time-related functionality in its own utility module. It would help to keep the main flathunt.py file readable.